docs/source/pipelines.rst

```diff
@@ -139,7 +139,7 @@ Pipeline Repositories
 
 Pipelines are structured as git repositories with a standardized directory hierarchy. The following
 example provides an overview of the pipeline repository structure.  It is adapted from the
-|MLflow Regression Pipeline repository|.
+|MLflow Recipe Regression Example repository|.
```

Comment:
Recipe -> Recipes, since we mostly name all MLflow products with plural (Projects, Models).